crates/cli/Cargo.toml

```
@@ -44,4 +44,13 @@ name = "reference"
 harness = false
 
 [features]
-vendored-openssl = ['openssl/vendored']
+default = ["rustls-tls"]
+
+rustls-tls = ["ureq/tls"]
+
+# Enables native-tls specific functionality not available by default.
+native-tls = ["ureq/native-tls"]
+native-tls-vendored = ["native-tls", "openssl/vendored"]
```

```suggestion
native-tls-vendored = ["native-tls", "native-tls/vendored"]
```
Unfortunately `ureq` doesn't re-expose this crate feature, but we should probably go as high as possible and not as low as possible. So let's just replace the `openssl` dependency with `native-tls`.

---

Great :)

---

In https://github.com/rustwasm/wasm-bindgen/pull/3511/commits/4cc4b2ee10fb2a0d02abbff1ce75d6cbec0617ec you removed the `vendored-openssl` crate feature, you can't really do that, that's a breaking change.

---

OK, I reverted the `vendored-openssl` feature.

---

oh wait a minute

---

Replaced two packages (features) like below.

1. `openssl` => `native-tls`
2. `vendored-openssl` => `openssl` + `native-tls/vendored`